src/Cpp/1-getting-started/1-HelloWindow/Input/Keyboard.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <cstdint>
+#include <set>
+
+// ReSharper disable once CppInconsistentNaming
```

Comment:
Comment seems out of place?

Reply:
Same Resharper vs clang-lint thing applies here, see above for discussion